packages/path_provider/path_provider/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
-## NEXT
+## 2.1.4
 
+* Adds Downloads directory support on Android
+* Updates minimum required path_provider_android to 2.2.0
```

Comment:
This is missing a period, which is required by the style guide linked from the PR checklist.

Reply:
Apologies, fixed now.